rio-backend/src/config/window.rs

```diff
@@ -57,6 +57,8 @@ pub struct Window {
     pub decorations: Decorations,
     #[serde(default = "bool::default", rename = "macos-use-unified-titlebar")]
     pub macos_use_unified_titlebar: bool,
+    #[serde(skip_serializing)]
+    pub title: Option<String>,
```

Comment:
can we rename it for `title_placeholder` or `initial_title` ?

Reply:
Sure! Will change